### PR TITLE
added welcome/hello.go

### PR DESCRIPTION
**Pull request by Girish-30**
- added hello.go
- created pelligent.in/hello module
- added launch configuration for
   - go file
   - go package
   - go test function
   - go test package

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module pelligent.in/main
```
Comment (vishal1201) [inline go.mod:1]: delete this mod file

**Review by vishal1201 — CHANGES_REQUESTED** [review]
resolves commented changes

File: go.mod (hunk 1 above)
Comment (Girish-30) [reply go.mod]: Deleted as requested in commit #138b54e002e540119f93f3b072ba2bcb4e60a352

**Review by vishalz-cognologix — APPROVED** [review]
Approved.

**Review by vishal1201 — APPROVED** [review]
Approved

**Comment by vishal1201** [comment]
Merged

**Pull request merged**